apps/riscv-tests/isa/macros/vector/vector_macros.h

```diff
@@ -73,7 +88,7 @@ int test_case;
     num_failed++;                                                       \
     return;                                                             \
   }                                                                     \
-  printf("PASSED.\n", casenum);                                         \
+  printf("PASSED.\n");                                         \
```

Comment:
Check the vertical alignment. Should we add the `casenum` in the `printf`  text?